roles/neutron-data/tasks/monitoring.yml

```diff
@@ -10,3 +10,10 @@
     name: "check-neutron-agents-duplicate"
     check: "{{ sensu_checks.neutron.check_neutron_agents_duplicate }}"
   notify: restart sensu-client missing ok
+
+- name: neutron router check multiple l3 agents active or all standby status
+  sensu_check_dict:
+    name: "check-neutron-l3-routers"
+    check: "{{ sensu_checks.neutron.check_neutron_l3_routers }}"
+  notify: restart sensu-client missing ok
+  when: inventory_hostname in groups['controller'][0]
```

Comment:
Need to add logic so the check is acitve on controller that has ucarp. So that, if controller[0] is down, check would still work on controller[1]. At any given time check should only do actual neutron calls on node that has ucarp